ui/src/containers/SampleViewContainer.jsx

```diff
@@ -57,22 +56,13 @@ class SampleViewContainer extends Component {
   }
 
   render() {
-    const { uiproperties, hardwareObjects } = this.props;
+    const { uiproperties } = this.props;
 
     if (!('sample_view' in uiproperties)) {
       return null;
     }
 
-    const motorUiProperties = uiproperties.sample_view.components.filter(
-      ({ value_type }) => value_type === 'MOTOR',
-    );
-
-    const motorhardwareObjects = Object.values(hardwareObjects).filter(
-      ({ type }) => type === 'MOTOR',
-    );
```

Comment:
`motorhardwareObjects` was then used in `MotorControls` to find the `sample_vertical` and `sample_horiztonal` hardware objects, which were then spread with the UI properties and passed down to `TwoAxisTranslationControl`. This was overly complex, especially because of `Object.values`.

Now I simply retrieve the two hardware objects in `TwoAxisTranslationControl` by their keys:

```ts
  const verticalMotor = useSelector(
    (state) => state.beamline.hardwareObjects[verticalMotorProps.attribute],
  );
  const horizontalMotor = useSelector(
    (state) => state.beamline.hardwareObjects[horizontalMotorProps.attribute],
  );
```